jwt.js

```diff
@@ -197,6 +198,12 @@ function fastifyJwt (fastify, options, next) {
     }
 
     let token
+    if (options.cookie && request.cookies[options.cookieName]) {
```

Comment:
`options` could very well be `null` or `undefined` or `[]` or `42`.

Reply:
Then it would throw `noAuthorizationInHeaderMessage`. Do you want me to enforce type checking if its an object in the if statement? I feel that if there is any other type of options the `noAuthorizationInHeaderMessage` error should be thrown (we assume that the user wanted to use the jwtVerify with no options) which is the default behaviour at the moment.

Reply:
I have to mention that I am using `fastify-cookie`.

Reply:
`fastify-cookie` has nothing to do with the options passed in to this plugin. This comment thread is in response to the changes you made in commit 1d6ee14.

Reply:
Ahh, I overlooked how the options object is being handled throughout, my bad.

Reply:
I have opted for a different approach where I don't touch the `options` object inside the `requestVerify` function. Instead, cookie is passed as an option to the plugin and the remaining logic remains the same.